diff --git a/libcxxabi/CMakeLists.txt b/libcxxabi/CMakeLists.txt
index a5afafd62c46..d3d0c2f68c49 100644
--- a/libcxxabi/CMakeLists.txt
+++ b/libcxxabi/CMakeLists.txt
@@ -76,7 +76,7 @@ MACRO_ENSURE_OUT_OF_SOURCE_BUILD(
 # Setup CMake Options
 #===============================================================================
 include(CMakeDependentOption)
-include(HandleCompilerRT)
+#include(HandleCompilerRT)
 
 # Define options.
 option(LIBCXXABI_ENABLE_EXCEPTIONS
diff --git a/libcxx/CMakeLists.txt b/libcxx/CMakeLists.txt
index 657714fb0094..49ec05b0212d 100644
--- a/libcxx/CMakeLists.txt
+++ b/libcxx/CMakeLists.txt
@@ -82,7 +82,7 @@ endif()
 # Setup CMake Options
 #===============================================================================
 include(CMakeDependentOption)
-include(HandleCompilerRT)
+#include(HandleCompilerRT)
 
 # Basic options ---------------------------------------------------------------
 option(LIBCXX_ENABLE_ASSERTIONS "Enable assertions independent of build mode." OFF)
diff --git a/libcxxabi/cmake/config-ix.cmake b/libcxxabi/cmake/config-ix.cmake
index 42cfd0a7d32c..336e161f0a7d 100644
--- a/libcxxabi/cmake/config-ix.cmake
+++ b/libcxxabi/cmake/config-ix.cmake
@@ -38,9 +38,9 @@ if (LIBCXXABI_SUPPORTS_NOSTDLIBXX_FLAG OR LIBCXXABI_SUPPORTS_NODEFAULTLIBS_FLAG)
     list(APPEND CMAKE_REQUIRED_LIBRARIES c)
   endif ()
   if (LIBCXXABI_USE_COMPILER_RT)
-    include(HandleCompilerRT)
-    find_compiler_rt_library(builtins LIBCXXABI_BUILTINS_LIBRARY
-                             FLAGS "${LIBCXXABI_COMPILE_FLAGS}")
+    #include(HandleCompilerRT)
+    #find_compiler_rt_library(builtins LIBCXXABI_BUILTINS_LIBRARY
+    #                         FLAGS "${LIBCXXABI_COMPILE_FLAGS}")
     list(APPEND CMAKE_REQUIRED_LIBRARIES "${LIBCXXABI_BUILTINS_LIBRARY}")
   else ()
     if (LIBCXXABI_HAS_GCC_S_LIB)
diff --git a/libcxxabi/src/CMakeLists.txt b/libcxxabi/src/CMakeLists.txt
index c237fb19f84f..792717841c9c 100644
--- a/libcxxabi/src/CMakeLists.txt
+++ b/libcxxabi/src/CMakeLists.txt
@@ -78,7 +78,7 @@ else()
 endif()
 
 if (LIBCXXABI_USE_COMPILER_RT)
-  find_compiler_rt_library(builtins LIBCXXABI_BUILTINS_LIBRARY)
+  #find_compiler_rt_library(builtins LIBCXXABI_BUILTINS_LIBRARY)
   list(APPEND LIBCXXABI_SHARED_LIBRARIES "${LIBCXXABI_BUILTINS_LIBRARY}")
 endif ()
 
@@ -136,7 +136,7 @@ if ( APPLE )
       message(WARNING "LLVM_USE_SANITIZER=${LLVM_USE_SANITIZER} is not supported on OS X")
     endif()
     if (LIBFILE)
-      find_compiler_rt_library(builtins LIBCXXABI_BUILTINS_LIBRARY)
+      #find_compiler_rt_library(builtins LIBCXXABI_BUILTINS_LIBRARY)
       get_filename_component(LIBDIR "${LIBCXXABI_BUILTINS_LIBRARY}" DIRECTORY)
       if (NOT IS_DIRECTORY "${LIBDIR}")
         message(FATAL_ERROR "Cannot find compiler-rt directory on OS X required for LLVM_USE_SANITIZER")
