diff --git a/llvm/CMakeLists.txt b/llvm/CMakeLists.txt
index 7083dd1331bb..b2360d540758 100644
--- a/llvm/CMakeLists.txt
+++ b/llvm/CMakeLists.txt
@@ -1203,8 +1203,8 @@ if (LLVM_INCLUDE_BENCHMARKS)
   set(BENCHMARK_ENABLE_GTEST_TESTS OFF CACHE BOOL "Disable Google Test in benchmark" FORCE)
   # Since LLVM requires C++11 it is safe to assume that std::regex is available.
   set(HAVE_STD_REGEX ON CACHE BOOL "OK" FORCE)
-  add_subdirectory(${LLVM_THIRD_PARTY_DIR}/benchmark 
-    ${CMAKE_CURRENT_BINARY_DIR}/third-party/benchmark)
+  #add_subdirectory(${LLVM_THIRD_PARTY_DIR}/benchmark 
+   # ${CMAKE_CURRENT_BINARY_DIR}/third-party/benchmark)
   add_subdirectory(benchmarks)
 endif()
 
